test/unity/toolkits/object_detection/test_object_detector.cxx

```diff
@@ -16,6 +16,8 @@
 #include <boost/test/unit_test.hpp>
 #include <core/util/test_macros.hpp>
 
+#include "../neural_net/neural_net_mocks.hpp"
```

Comment:
Are we using double quotes with a relative path here because the header we're importing is part of the same project, i.e. `test`?